OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseCombinedInfoFragment.java

```diff
@@ -134,7 +134,7 @@ public void onActivityCreated(Bundle savedInstanceState) {
                 }
             }
 
-            if (environment.getConfig().isNotificationEnabled()
```

Comment:
@staubina This check needs to be look into depth, its getting controlled with 'PUSH_NOTIFICATIONS' feature flag, it needs to look into its history, what this settings actually do and how we can connect it with our FCM integration.

Reply:
This looks like it is related to the old notifications framework.  I do not think we use this in production and it also looks like dead code.  I am planning to remove this in my next push. Including the UI it mentions.